tests/actions/EnforceActionExportRestrictions.ts

```diff
@@ -27,6 +29,11 @@ describe('ReportUtils', () => {
         // @ts-expect-error the test is asserting that it's undefined, so the TS error is normal
         expect(ReportUtils.getAllReportActions).toBeUndefined();
     });
+
+    it('does not export getReport', () => {
+        // @ts-expect-error the test is asserting that it's undefined, so the TS error is normal
+        expect(ReportUtils.getReport).toBeUndefined();
+    });
```

Comment:
Let's rename `ReportUtils.getReport` to `ReportUtils. getReportOrDraftReport` and update the test